### ISSUE TITLE
Oculus Go Update?

### ISSUE THREAD

**Issue by ClonedPuppy** [mitchemmc/UE4VRCookbook#3]
Hi Mitch, going through your excellent book. Thank you for writing this. I'm slowly working my way through the chapters, and had some issues in the earlier chapter 2, where you are explaining how to work around the back button of the gearvr. I guess the oculus go has some differences in how it works in regards to these buttons, but after a lot of searching on the net I can't really reach any explanation to why it doesn't work. So, finally to my question, will you be updating any of these examples, or the book for the Oculus GO?

**Comment by mitchemmc** [comment]
Hey, I don't currently have an Oculus Go to test on, but I may be able to get my hands on one in the next couple of weeks to debug this issue. I'll post an update here after I get to test it.

**Comment by ClonedPuppy** [comment]
Thanks for your reply Mitch, I will keep digging for info on oculus dev site, too bad their unreal / oculus go documentation is so incomplete.

**Comment by ClonedPuppy** [comment]
I think this might be the issue why the back button blueprint you start the book with is not working.
https://forums.oculusvr.com/developer/discussion/65429/oculus-go-back-button-long-press-issues

**Comment by mitchemmc** [comment]
Alright I got to test an Oculus Go.
Back with the first GearVR (before the external controller was added) there was one button on the headset to control both going back and taking users to the Oculus menu. 
Therefore it was necessary to override this behaviour to allow a single click of this button to interact with your app while ensuring a long click still opened the Oculus menu. However, later versions of the GearVR and the Oculus Go make use of both a Back button as well as a separate Home button.
This change in hardware meant that [Oculus stopped allowing](https://developer.oculus.com/documentation/mobilesdk/latest/concepts/mobile-umenu-intro/) the Home button to be overridden in order to ensure it always behaved in the same manner. So this means that you want to only tie actions to the back button which is mapped to the[ Motion Controller Face Button 1](https://developer.oculus.com/documentation/unreal/latest/concepts/unreal-controller-input-mapping-reference/#controller-input-mapping-reference-fig-9) for UE4.
Also note using a long press of the back button may still be reserved with the GearVR as noted in the reserved user interactions doc linked above, I don't have the latest GearVR to test that on though, sorry.
